src/System.ValueTuple/src/System/Numerics/Hashing/Hash.cs

```diff
@@ -0,0 +1,101 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+
+namespace System.Numerics.Hashing
+{
+    internal static class Hash
```

Comment:
I kept it since it saves on lines of code in `ValueTuple`; if we didn't have these functions we'd have to write something like

```cs
int result = EqualityComparer<T1>.Default.GetHashCode(Item1);
result = HashHelpers.Combine(result, EqualityComparer<T2>.Default.GetHashCode(Item2));
result = HashHelpers.Combine(result, EqualityComparer<T3>.Default.GetHashCode(Item3));
...
return result;
```

which is significantly longer than just `Hash.CombineHashCodes(Item1, Item2, Item3, ...)`.